services/simcore/docker-compose.yml

```diff
@@ -607,6 +658,10 @@ services:
         delay: 5s
         max_attempts: 3
         window: 120s
+      placement:
+        constraints:
+          - node.role == manager  # safety measure. traefik must be deployed on manager nodes
```

Comment:
same I guess this constraint is already in simcore. but it's ok to add it again.

Reply:
removed